src/Microsoft.Net.Http.Headers/CacheControlHeaderValue.cs

```diff
@@ -11,18 +11,18 @@ namespace Microsoft.Net.Http.Headers
 {
     public class CacheControlHeaderValue
     {
-        private const string MaxAgeString = "max-age";
-        private const string MaxStaleString = "max-stale";
-        private const string MinFreshString = "min-fresh";
-        private const string MustRevalidateString = "must-revalidate";
-        private const string NoCacheString = "no-cache";
-        private const string NoStoreString = "no-store";
-        private const string NoTransformString = "no-transform";
-        private const string OnlyIfCachedString = "only-if-cached";
-        private const string PrivateString = "private";
-        private const string ProxyRevalidateString = "proxy-revalidate";
-        private const string PublicString = "public";
-        private const string SharedMaxAgeString = "s-maxage";
```

Comment:
I don't know if David will be OK with this because of the switch statement. Like I said, I'm fine with consts given we don't mess up the values, but I would like consistency.